plugins/generator-1.16.4/forge-1.16.4/procedures/get_localized_text.java.ftl

```diff
@@ -1 +1 @@
-(new TranslationTextComponent(${input$key}).getUnformattedComponentText())
\ No newline at end of file
+(new TranslationTextComponent(${input$key}).getString())
```

Comment:
This change may be needed in 1.15.2, please test in 1.15.2 and fix is that is correct, then I will merge

Reply:
I just tested in 1.15.2 and it works fine with `.getUnformattedComponentText()`

Reply:
It will probably ignore formatting codes, though, so getString may be a better choice

Reply:
Done. Waiting for a final review.